eno-core/src/main/java/fr/insee/eno/core/processing/out/steps/lunatic/LunaticAddControlFormat.java

```diff
@@ -14,6 +15,7 @@
  * Processing adding format controls to components
  * Format controls are controls generated by the min/max/decimals attributes of the Datepicker/InputNumber components
  */
+@Slf4j
```

Comment:
I don't see any log here.
I think you can remove `@Slf4j` annotations or add some log

Reply:
line 198

Mais à ce sujet il faut que je configure l'implem de logger dans les tests de la lib pour que le warning à cette ligne n'apparaisse pas quand je teste le cas où on entre dans le bloc if.

Et d'ailleurs plus globalement sur le logging c'est un sujet que je procrastine dans Eno : il faudrait faire une revue de ce qui est logué dans les cas d'usage courants et vérifier qu'on a les bonnes infos (pas trop / pas trop peu / à peu près au même niveau de précision) sur les logs en `log.info`

Reply:
Je vais traiter ces sujets dans une prochaine PR de refacto sur le logging